client/src/index.css

```diff
@@ -136,6 +136,8 @@ video {
   border-radius: 0.2rem;
   background-color: var(--bg);
   border: 0.1rem solid var(--border);
+  /* Context menu is a layer above all other elements, but below dialog (see Dialog's CSS classes) */
+  z-index: 1;
```

Comment:
I believe I removed `z-index` for context menu in #150 due to some conflicts with the help dialog and the video element for demo, but I think this should be the right solution. 


I don't know any other way to solve these issues. 

As far as I can tell, we have 3 layers: 
- Normal elements
- Context Menu
- Dialog 

Let me know if you can think of any edge cases.